lang/en.json

```
@@ -294,7 +294,7 @@
   "collective.editHost.withdrawApp": "Are you sure you want to withdraw your application to {name}?",
   "collective.emptyBalance.body": "Are you sure you want to {action} to {collective}?",
   "collective.emptyBalance.header": "{action} Balance",
-  "collective.event.delete.isNotDeletable-message": "{type, select, EVENT {Events} PROJECT {Projects}} with transactions, orders or paid expenses cannot be deleted. Please archive it instead.",
+  "collective.event.delete.isNotDeletable-message": "{type, select, EVENT {Events} PROJECT {Projects} other {}} with transactions, orders or paid expenses cannot be deleted. Please archive it instead.",
```

A default string for this one could actually be nice

```suggestion
  "collective.event.delete.isNotDeletable-message": "{type, select, EVENT {Events} PROJECT {Projects} other {Accounts}} with transactions, orders or paid expenses cannot be deleted. Please archive it instead.",
```